src/pages/dashboard/application_info/index.tsx

```diff
@@ -24,7 +24,7 @@ const RecentApplication: React.FC = () => {
 										<ApplicationItem
 											key={application._id}
 											studentID={application.studentId}
-											studentName={application.name}
+											studentName={application.name.split(" ")[0] + " " + application.name.split(" ")[1] + " ..."}
```

Comment:
There is nothing wrong with the code, but there is another simple way to do this

`studentName={application.name.split(" ").slice(0, 2).join(" ") + " ..."};`

https://www.w3schools.com/jsref/jsref_slice_array.asp